docs/data-guidelines.md

```diff
@@ -174,6 +174,6 @@ This versioning scheme came at [Apple's request, in #2006](https://github.com/md
 Features can be removed from BCD if it is considered irrelevant. A feature can be considered irrelevant if any of these conditions are met:
 
 - a feature was never implemented in any browser and the specification has been abandoned.
-- a feature was implemented and has since been removed from all browser releases dating back 2 years.
+- a feature was implemented and has since been removed from all majors browsers dating back two or more years ago, or is unsupported in all major browsers released in the past five years.
```

Comment:
Is it OK if we do this as a separate bullet? I think this simplifies the idea here a little bit better.

```suggestion
- a feature was implemented and has since been removed from all browsers dating back two or more years ago.
- a feature is unsupported in all releases in the past five years.
```

I would interpret this to mean that we can remove:

* Any unimplemented, abandoned feature now
* Any feature which has been dropped from all browsers (including IE) two years after the last browser in our data set drops it
* Any feature which is unsupported by all browsers in our data set that have had a release <5 years ago (e.g., IE right now, or in 4.5 years, any EdgeHTML-only features)

Reply:
Yep, that is the idea exactly. I think having the separate bullet is better, thank you! :+1: